src/main/script/indexMongoCollections.cmd

```diff
@@ -0,0 +1,5 @@
+@REM Place this script on mongoDB installation directory. It runs an instance of mongo daemon and a .js script that adds indexes for xlimeres. Change --dbpath arguments if needed.
+@echo Connecting to mongo
+START "Connecting" mongod.exe --dbpath "C:\Program Files\MongoDB\data"
```

Comment:
This launches a mongo server (i.e. assumes that mongod is not currently running on the machine), and it hardcodes the path of where the data will be stored. Remove the line and add a comment that we expect mongod to be running on the machine.